test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/http/RequestParamsBuilder.java

```diff
@@ -21,6 +21,12 @@ private RequestParamsBuilder(final String url, final String mediaType) {
         this.url = url;
         this.mediaType = mediaType;
     }
+    
+    private RequestParamsBuilder(final String url, final String mediaType, MultivaluedMap<String, Object> headers ) {
+        this.url = url;
+        this.mediaType = mediaType;
+        this.headers=headers;
```

Comment:
spacing formatting, around = sign. Also looks like there is a space at the end of the method params